main.webapp/src/test/java/org/starchartlabs/river/test/util/StarchartLinkExtractor.java

```diff
@@ -0,0 +1,65 @@
+package org.starchartlabs.river.test.util;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.restdocs.hypermedia.Link;
+import org.springframework.restdocs.hypermedia.LinkExtractor;
+import org.springframework.restdocs.operation.OperationResponse;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
```

Comment:
Probably worth doc'ing here that you'd like to reduce this once spring allows, and link to the PR you made. Possibly also include a TODO